linux_os/guide/system/accounts/accounts-restrictions/accounts_authorized_local_users/ansible/shared.yml

```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_sle
+# platform = multi_platform_sle,multi_platform_rhel
 # reboot = false
 # strategy = restrict
 # complexity = low
```

Comment:
This seem low to me, do we want to raise it to medium or high? Looking for more guidance on this.

Reply:
I don't know how to answer this. Following the documentation [here](https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html?highlight=complexity#remediations) it indicates it's just informative and there is no real impact on anything. This is translated into ansible tags but I doubt people are actually using this meta information in any way.

```
The estimated complexity or difficulty of applying the fix to the target. Only informative for now
```

Reply:
My issue is that without tailoring this rule will cause issues when remediated. Setting `disruption` to medium or high might be good idea or add warning like in some other rules.

Reply:
I agree. You can add a warning to the rule.yml similar as it's done in: https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/system/software/sudo/sudo_dedicated_group/rule.yml#L14

Reply:
The more I think about it, just deleting users doesn't sit right with me, do want to disable remediation by default?

Reply:
> The more I think about it, just deleting users doesn't sit right with me, do want to disable remediation by default?

Why?

Reply:
By default this will delete all user added local users on the box, for a new system that is fine, but for existing systems this will be highly distributive.

Reply:
OK, so please remove it. Also, please make sure that the test scenarios will pass.